src/Core/Grid/Presenter/GridPresenter.php

```diff
@@ -137,8 +137,7 @@ public function getOrderingPosition(GridInterface $grid)
         /** @var ColumnInterface $column */
         foreach ($grid->getDefinition()->getColumns() as $column) {
             if ($column instanceof PositionColumn &&
-                strtolower($column->getId()) == strtolower($searchCriteria->getOrderBy()) &&
-                'asc' == strtolower($searchCriteria->getOrderWay())
+                strtolower($column->getId()) == strtolower($searchCriteria->getOrderBy())
```

Comment:
I disable the condition that removed the drag-and-drop when position sorting direction was not ASC